iOS/CodiumUnWrapper/CodiumUnWrapper.swift

```diff
@@ -0,0 +1,155 @@
+// Codium UnWrapper
```

Comment:
Could you change it to `Safe Optional Unwrap` I tried to find the word "Unwrapper" but seem like it's not a word.

Reply:
We don't need the name CODIUM, since this is in CODIUM Repo

Reply:
actually when users see the CODIUM name they will not forget it, in my opinion nobody can remember where they found the codes but every body remember the name of the class or library such as Alamofire or KingFisher or ...  ,and also Safe Optional Unwrap is not a unique name which other people can remember the specific name , that's why I named it CODIUM UnWrapper
and for UnWrapper you can see this link below 
https://en.wiktionary.org/wiki/unwrapper

Reply:
next step is make it as a pod so the users will install `CODIUM UnWrapper` pod in their projects